unlock-app/src/services/storageService.js

```diff
@@ -53,19 +53,25 @@ export class StorageService extends EventEmitter {
    * @param {*} senderAddress
    * @param {*} recipientAddress
    * @param {*} chain
+   * @paran {*} data (input of transaction, optional)
    */
   async storeTransaction(
     transactionHash,
     senderAddress,
     recipientAddress,
-    chain
+    chain,
+    beneficiaryAddress /** Used in the context of key purchase *for* */,
+    data
```

Comment:
This was added because the paywall currently save that, so I assumed this was useful... maybe it won't but at least its there if we need it!